doc/whats_new.rst

```diff
@@ -233,8 +233,13 @@ Enhancements
    - Added new return type ``(data, target)`` : tuple option to
      :func:`load_iris` dataset, 
      (`#7049 <https://github.com/scikit-learn/scikit-learn/pull/7049>`_)
```

Comment:
Is this better ?
![screenshot from 2016-08-06 17-34-34](https://cloud.githubusercontent.com/assets/12952080/17456628/19051b5e-5bfc-11e6-83d8-2a714ce5035a.png)